jwcrypto/jwk.py

```diff
@@ -79,10 +87,12 @@ class ParmType(Enum):
 }
 """Regstry of valid key parameters"""
 
-# RFC 7518 - 7.6
+# RFC 7518 - 7.6 , RFC 8037 - 5
 JWKEllipticCurveRegistry = {'P-256': 'P-256 curve',
                             'P-384': 'P-384 curve',
-                            'P-521': 'P-521 curve'}
+                            'P-521': 'P-521 curve',
+                            'Ed25519': 'Ed25519 signature algorithm key pairs',
```

Comment:
This should probably just be: "Ed25519 curve"

Reply:
Or maybe better: "Edwards-curve 25519 for Digital Signatures"

Reply:
Thanks - I took that directly from section 5 of RFC 8037:

```
   The following has been added to the "JSON Web Key Elliptic Curve"
   registry:

   o  Curve Name: "Ed25519"
   o  Curve Description: Ed25519 signature algorithm key pairs
```
Am happy either way.

Reply:
Ah, I think the RFC wording is a bit unfortunate, but ok